Fails to build pyjnius

<!--
The issue tracker is a tool to address bugs NOT a support platform.
Please use the Discord community or Stack Overflow for support questions,
more information at https://github.com/kivy/python-for-android#support
-->

### Checklist

- [x] the issue is indeed a bug and not a support request
- [x] issue doesn't already exist: https://github.com/kivy/python-for-android/issues
- [x] I have a short, runnable example that reproduces the issue
- [x] I reproduced the problem with the latest development version (`p4a.branch = develop`)
- [ ] I used the grave accent (aka backticks) to format code or logs when appropriated

### Versions

- Python: `3.11.5`
- OS: `Arch Linux x86_64`
- Kivy: `v2.2.1`
- Cython: `3.0.4`
- OpenJDK: `1.8.0_392`

### Description

I was trying to compile a minimal example and it failed when was building `sdl2`

***main.py:***
```python
import kivy

from kivy.app import App
from kivy.uix.button import Button

class DummyApp(App):
    def build(self):
        return Button(text="Hello World")

if __name__ == '__main__':
    DummyApp().run()
```

***Command:***
```sh
p4a apk --private ./tst_android_kivy/main.py --package=org.example.myapp --name "My application" --version 0.1 --bootstrap=sdl2 --requirements=python3,kivy --arch=arm64-v8a --sdk-dir=/opt/android-sdk/ --android_api=27 --ndk_dir=/opt/android-ndk
```

### Logs

```terminal
➜  tst_android_kivy p4a apk --private ./tst_android_kivy/main.py --package=org.example.myapp --name "My application" --version 0.1 --bootstrap=sdl2 --requirements=python3,kivy --arch=arm64-v8a --sdk-dir=/opt/android-sdk/ --android_api=27 --ndk_dir=/opt/android-ndk
[WARNING]: prerequisites.py is experimental and does not support all prerequisites yet.
[WARNING]: Please report any issues to the python-for-android issue tracker.
[INFO]:    Will compile for the following archs: arm64-v8a
[INFO]:    Getting Android API version from user argument: 27
[WARNING]: Target API 27 < 30
[WARNING]: Target APIs lower than 30 are no longer supported on Google Play, and are not recommended. Note that the Target API can be higher than your device Android version, and should usually be as high as possible.
[INFO]:    Available Android APIs are (27)
[INFO]:    Requested API target 27 is available, continuing.
[INFO]:    Getting NDK dir from from user argument
[INFO]:    Found NDK version 26b
[WARNING]: Maximum recommended NDK version is 25b, but newer versions may work.
[WARNING]: Newer NDKs may not be fully supported by p4a.
[WARNING]: NDK API target was not set manually, using the default of 21 = min(android-api=27, default ndk-api=21)
[INFO]:    ccache is missing, the build will not be optimized in the future.
[INFO]:    No existing dists meet the given requirements!
[INFO]:    No dist exists that meets your requirements, so one will be built.
[INFO]:    Found a single valid recipe set: ['hostpython3', 'libffi', 'openssl', 'sdl2_image', 'sdl2_mixer', 'sdl2_ttf', 'sqlite3', 'python3', 'sdl2', 'setuptools', 'six', 'pyjnius', 'android', 'kivy']
[INFO]:    The selected bootstrap is sdl2
[INFO]:    # Creating dist with sdl2 bootstrap
[INFO]:    Dist will have name unnamed_dist_8 and requirements (python3, kivy)
[INFO]:    Dist contains the following requirements as recipes: ['hostpython3', 'libffi', 'openssl', 'sdl2_image', 'sdl2_mixer', 'sdl2_ttf', 'sqlite3', 'python3', 'sdl2', 'setuptools', 'six', 'pyjnius', 'android', 'kivy']
[INFO]:    Dist will also contain modules (requests, urllib3, idna, certifi, chardet) installed from pip
[INFO]:    Dist will be build in mode debug
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Recipe build order is ['hostpython3', 'libffi', 'openssl', 'sdl2_image', 'sdl2_mixer', 'sdl2_ttf', 'sqlite3', 'python3', 'sdl2', 'setuptools', 'six', 'pyjnius', 'android', 'kivy']
[INFO]:    The requirements (certifi, chardet, idna, requests, urllib3) were not found as recipes, they will be installed with pip.
[INFO]:    # Downloading recipes
[INFO]:    Downloading hostpython3
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/packages/hostpython3
[INFO]:    -> running basename https://www.python.org/ftp/python/3.10.10/Python-3.10.10.tgz
[INFO]:    hostpython3 download already cached, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Downloading libffi
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/packages/libffi
[INFO]:    -> running basename https://github.com/libffi/libffi/archive/v3.4.2.tar.gz
[INFO]:    libffi download already cached, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Downloading openssl
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/packages/openssl
[INFO]:    -> running basename https://www.openssl.org/source/openssl-1.1.1m.tar.gz
[INFO]:    openssl download already cached, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Downloading sdl2_image
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/packages/sdl2_image
[INFO]:    -> running basename https://github.com/libsdl-org/SDL_image/releases/download/release-2.6.2/SDL2_image-2.6.2.tar.gz
[INFO]:    sdl2_image download already cached, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Downloading sdl2_mixer
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/packages/sdl2_mixer
[INFO]:    -> running basename https://github.com/libsdl-org/SDL_mixer/releases/download/release-2.6.2/SDL2_mixer-2.6.2.tar.gz
[INFO]:    sdl2_mixer download already cached, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Downloading sdl2_ttf
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/packages/sdl2_ttf
[INFO]:    -> running basename https://github.com/libsdl-org/SDL_ttf/releases/download/release-2.20.1/SDL2_ttf-2.20.1.tar.gz
[INFO]:    sdl2_ttf download already cached, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Downloading sqlite3
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/packages/sqlite3
[INFO]:    -> running basename https://www.sqlite.org/2021/sqlite-amalgamation-3350500.zip
[INFO]:    sqlite3 download already cached, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Downloading python3
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/packages/python3
[INFO]:    -> running basename https://www.python.org/ftp/python/3.10.10/Python-3.10.10.tgz
[INFO]:    python3 download already cached, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Downloading sdl2
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/packages/sdl2
[INFO]:    -> running basename https://github.com/libsdl-org/SDL/releases/download/release-2.26.1/SDL2-2.26.1.tar.gz
[INFO]:    sdl2 download already cached, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Downloading setuptools
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/packages/setuptools
[INFO]:    -> running basename https://pypi.python.org/packages/source/s/setuptools/setuptools-51.3.3.tar.gz
[INFO]:    setuptools download already cached, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Downloading six
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/packages/six
[INFO]:    -> running basename https://pypi.python.org/packages/source/s/six/six-1.15.0.tar.gz
[INFO]:    six download already cached, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Downloading pyjnius
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/packages/pyjnius
[INFO]:    -> running basename https://github.com/kivy/pyjnius/archive/1.5.0.zip
[INFO]:    pyjnius download already cached, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Downloading android
[INFO]:    Skipping android download as no URL is set
[INFO]:    Downloading kivy
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/packages/kivy
[INFO]:    -> running basename https://github.com/kivy/kivy/archive/2.2.1.zip
[INFO]:    kivy download already cached, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    # Building all recipes for arch arm64-v8a
[INFO]:    # Unpacking recipes
[INFO]:    Unpacking hostpython3 for arm64-v8a
[INFO]:    -> running basename https://www.python.org/ftp/python/3.10.10/Python-3.10.10.tgz
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/build/other_builds/hostpython3/desktop
[INFO]:    hostpython3 is already unpacked, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Unpacking libffi for arm64-v8a
[INFO]:    -> running basename https://github.com/libffi/libffi/archive/v3.4.2.tar.gz
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/build/other_builds/libffi/arm64-v8a__ndk_target_21
[INFO]:    libffi is already unpacked, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Unpacking openssl for arm64-v8a
[INFO]:    -> running basename https://www.openssl.org/source/openssl-1.1.1m.tar.gz
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/build/other_builds/openssl/arm64-v8a__ndk_target_21
[INFO]:    openssl is already unpacked, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Unpacking sdl2_image for arm64-v8a
[INFO]:    -> running basename https://github.com/libsdl-org/SDL_image/releases/download/release-2.6.2/SDL2_image-2.6.2.tar.gz
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni
[INFO]:    sdl2_image is already unpacked, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Unpacking sdl2_mixer for arm64-v8a
[INFO]:    -> running basename https://github.com/libsdl-org/SDL_mixer/releases/download/release-2.6.2/SDL2_mixer-2.6.2.tar.gz
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni
[INFO]:    sdl2_mixer is already unpacked, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Unpacking sdl2_ttf for arm64-v8a
[INFO]:    -> running basename https://github.com/libsdl-org/SDL_ttf/releases/download/release-2.20.1/SDL2_ttf-2.20.1.tar.gz
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni
[INFO]:    sdl2_ttf is already unpacked, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Unpacking sqlite3 for arm64-v8a
[INFO]:    -> running basename https://www.sqlite.org/2021/sqlite-amalgamation-3350500.zip
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/build/other_builds/sqlite3/arm64-v8a__ndk_target_21
[INFO]:    sqlite3 is already unpacked, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Unpacking python3 for arm64-v8a
[INFO]:    -> running basename https://www.python.org/ftp/python/3.10.10/Python-3.10.10.tgz
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/build/other_builds/python3/arm64-v8a__ndk_target_21
[INFO]:    python3 is already unpacked, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Unpacking sdl2 for arm64-v8a
[INFO]:    -> running basename https://github.com/libsdl-org/SDL/releases/download/release-2.26.1/SDL2-2.26.1.tar.gz
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni
[INFO]:    sdl2 is already unpacked, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Unpacking setuptools for arm64-v8a
[INFO]:    -> running basename https://pypi.python.org/packages/source/s/setuptools/setuptools-51.3.3.tar.gz
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/build/other_builds/setuptools/arm64-v8a__ndk_target_21
[INFO]:    setuptools is already unpacked, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Unpacking six for arm64-v8a
[INFO]:    -> running basename https://pypi.python.org/packages/source/s/six/six-1.15.0.tar.gz
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/build/other_builds/six/arm64-v8a__ndk_target_21
[INFO]:    six is already unpacked, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Unpacking pyjnius for arm64-v8a
[INFO]:    -> running basename https://github.com/kivy/pyjnius/archive/1.5.0.zip
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/build/other_builds/pyjnius-sdl2/arm64-v8a__ndk_target_21
[INFO]:    pyjnius is already unpacked, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    -> running cp -a /home/xou/.local/lib/python3.11/site-packages/pythonforandroid/recipes/android/src /home/xou/.local/share/py...(and 81 more)
[INFO]:    Unpacking kivy for arm64-v8a
[INFO]:    -> running basename https://github.com/kivy/kivy/archive/2.2.1.zip
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/build/other_builds/kivy/arm64-v8a__ndk_target_21
[INFO]:    kivy is already unpacked, skipping
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    # Prebuilding recipes
[INFO]:    Prebuilding hostpython3 for arm64-v8a
[INFO]:    hostpython3 has no prebuild_arm64_v8a, skipping
[INFO]:    Applying patches for hostpython3[arm64-v8a]
[INFO]:    hostpython3 already patched, skipping
[INFO]:    Prebuilding libffi for arm64-v8a
[INFO]:    libffi has no prebuild_arm64_v8a, skipping
[INFO]:    Applying patches for libffi[arm64-v8a]
[INFO]:    libffi already patched, skipping
[INFO]:    Prebuilding openssl for arm64-v8a
[INFO]:    openssl has no prebuild_arm64_v8a, skipping
[INFO]:    Prebuilding sdl2_image for arm64-v8a
[INFO]:    sdl2_image has no prebuild_arm64_v8a, skipping
[INFO]:    Applying patches for sdl2_image[arm64-v8a]
[INFO]:    sdl2_image already patched, skipping
[INFO]:    Prebuilding sdl2_mixer for arm64-v8a
[INFO]:    sdl2_mixer has no prebuild_arm64_v8a, skipping
[INFO]:    Prebuilding sdl2_ttf for arm64-v8a
[INFO]:    sdl2_ttf has no prebuild_arm64_v8a, skipping
[INFO]:    Prebuilding sqlite3 for arm64-v8a
[INFO]:    sqlite3 has no prebuild_arm64_v8a, skipping
[INFO]:    Prebuilding python3 for arm64-v8a
[INFO]:    python3 has no prebuild_arm64_v8a, skipping
[INFO]:    Applying patches for python3[arm64-v8a]
[INFO]:    python3 already patched, skipping
[INFO]:    Prebuilding sdl2 for arm64-v8a
[INFO]:    sdl2 has no prebuild_arm64_v8a, skipping
[INFO]:    Applying patches for sdl2[arm64-v8a]
[INFO]:    sdl2 already patched, skipping
[INFO]:    Prebuilding setuptools for arm64-v8a
[INFO]:    setuptools has no prebuild_arm64_v8a, skipping
[INFO]:    Prebuilding six for arm64-v8a
[INFO]:    six has no prebuild_arm64_v8a, skipping
[INFO]:    Prebuilding pyjnius for arm64-v8a
[INFO]:    pyjnius has no prebuild_arm64_v8a, skipping
[INFO]:    Applying patches for pyjnius[arm64-v8a]
[INFO]:    pyjnius already patched, skipping
[INFO]:    Prebuilding android for arm64-v8a
[INFO]:    android has no prebuild_arm64_v8a, skipping
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/build/other_builds/android-sdl2/arm64-v8a__ndk_target_21/android
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Prebuilding kivy for arm64-v8a
[INFO]:    kivy has no prebuild_arm64_v8a, skipping
[INFO]:    Applying patches for kivy[arm64-v8a]
[INFO]:    kivy already patched, skipping
[INFO]:    # Building recipes
[INFO]:    Building hostpython3 for arm64-v8a
[INFO]:    hostpython3 said it is already built, skipping
[INFO]:    Building libffi for arm64-v8a
[INFO]:    libffi said it is already built, skipping
[INFO]:    -> running cp /home/xou/.local/share/python-for-android/build/other_builds/libffi/arm64-v8a__ndk_target_21/libffi/.libs/libff...(and 94 more)
[INFO]:    Building openssl for arm64-v8a
[INFO]:    openssl said it is already built, skipping
[INFO]:    -> running cp /home/xou/.local/share/python-for-android/build/other_builds/openssl/arm64-v8a__ndk_target_21/openssl1.1/libcry...(and 217 more)
[INFO]:    Building sdl2_image for arm64-v8a
[INFO]:    Building sdl2_mixer for arm64-v8a
[INFO]:    Building sdl2_ttf for arm64-v8a
[INFO]:    Building sqlite3 for arm64-v8a
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/build/other_builds/sqlite3/arm64-v8a__ndk_target_21/sqlite3
[INFO]:    -> running ndk-build V=1 NDK_DEBUG=1 APP_PLATFORM=android-21 APP_ABI=arm64-v8a
[INFO]:    <- directory context /home/xou/Desktop/xou/programming/python/trash/tst_android_kivy
[INFO]:    Building python3 for arm64-v8a
[INFO]:    python3 said it is already built, skipping
[INFO]:    Building sdl2 for arm64-v8a
[INFO]:    -> directory context /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni
[INFO]:    -> running ndk-build V=1 NDK_DEBUG=1
           working: make: *** [/opt/android-ndk/build/core/build-binary.mk:415: /home/xou/.local/share/python-for-android/build/bootstrap_builds/...(and 65 more)  Exception in thread background thread for pid 95924:
Traceback (most recent call last):
  File "/usr/lib/python3.11/threading.py", line 1038, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.11/threading.py", line 975, in run
    self._target(*self._args, **self._kwargs)
  File "/home/xou/.local/lib/python3.11/site-packages/sh.py", line 1641, in wrap
    fn(*rgs, **kwargs)
  File "/home/xou/.local/lib/python3.11/site-packages/sh.py", line 2569, in background_thread
    handle_exit_code(exit_code)
  File "/home/xou/.local/lib/python3.11/site-packages/sh.py", line 2269, in fn
    return self.command.handle_command_exit_code(exit_code)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/xou/.local/lib/python3.11/site-packages/sh.py", line 869, in handle_command_exit_code
    raise exc
sh.ErrorReturnCode_2:

  RAN: /opt/android-ndk/ndk-build V=1 NDK_DEBUG=1

  STDOUT:
Android NDK: WARNING:/home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni/application/src/Android.mk:main: non-system libraries in linker flags: -lpython3.10
Android NDK:     This is likely to result in incorrect builds. Try using LOCAL_STATIC_LIBRARIES
Android NDK:     or LOCAL_SHARED_LIBRARIES instead to list the library dependencies of the
Android NDK:     current module
/opt/android-ndk/build/core/build-binary.mk:583: Android NDK: Module main depends on undefined modules: python_shared
rm -f /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/libs/arm64-v8a/*
[arm64-v8a] Compile        : SDL2 <= SDL.c
rm -f /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/obj/local/arm64-v8a/objs-debug/SDL2/src/SDL.o
/opt/android-ndk/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -MMD -MP -MF /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/obj/local/arm64-v8a/objs-debug/SDL2/src/SDL.o.d -target aarch64-none-linux-android21 -fdata-sections -ffunction-sections -fstack-protector-strong -funwind-tables -no-canonical-prefixes  --sysroot /opt/android-ndk/toolchains/llvm/prebuilt/linux-x86_64/sysroot -g -Wno-invalid-command-line-argument -Wno-unused-command-line-argument  -D_FORTIFY_SOURCE=2 -fpic -O0 -UNDEBUG -fno-limit-debug-info  -I/home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni/SDL/include -I/opt/android-ndk/sources/android/cpufeatures -I/home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni/SDL   -DANDROID -DGL_GLEXT_PROTOTYPES -Wall -Wextra -Wdocumentation -Wdocumentation-unknown-command -Wmissing-prototypes -Wunreachable-code-break -Wunneeded-internal-declaration -Wmissing-variable-declarations -Wfloat-conversion -Wshorten-64-to-32 -Wunreachable-code-return -Wshift-sign-overflow -Wstrict-prototypes -Wkeyword-macro -Wno-unused-parameter -Wno-sign-compare -Wformat -Werror=format-security  -c  /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni/SDL/src/SDL.c -o /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/obj/local/arm64-v8a/objs-debug/SDL2/src/SDL.o
[arm64-v8a] Compile        : SDL2 <= SDL_assert.c
rm -f /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/obj/local/arm64-v8a/objs-debug/SDL2/src/SDL_assert.o
/opt/android-ndk/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -MMD -MP -MF /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/obj/local/arm64-v8a/objs-debug/SDL2/src/SDL_assert.o.d -target aarch64-none-linux-android21 -fdata-sections -ffunction-sections -fstack-protector-strong -funwind-tables -no-canonical-prefixes  --sysroot /opt/android-ndk/toolchains/llvm/prebuilt/linux-x86_64/sysroot -g -Wno-invalid-command-line-argument -Wno-unused-command-line-argument  -D_FORTIFY_SOURCE=2 -fpic -O0 -UNDEBUG -fno-limit-debug-info  -I/home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni/SDL/include -I/opt/android-ndk/sources/android/cpufeatures -I/home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni/SDL   -DANDROID -DGL_GLEXT_PROTOTYPES -Wall -Wextra -Wdocumentation -Wdocumentation-unknown-command -Wmissing-prototypes -Wunreachable-code-break -Wunneeded-internal-declaration -Wmissing-variable-declarations -Wfloat-conversion -Wshorten-64-to-32 -Wunreachable-code-return -Wshift-sign-overflow -Wstrict-prototypes -Wkeyword-macro -Wno-unused-parameter -Wno-sign-compare -Wformat -Werror=format-security  -c  /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni/SDL/src/SDL_assert.c -o /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/obj/local/arm64-v8a/objs-debug/SDL2/src/SDL_assert.o
[arm64-v8a] Compile        : SDL2 <= SDL_dataqueue.c
rm -f /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/obj/local/arm64-v8a/objs-debug/SDL2/src/SDL_dataqueue.o
/opt/android-ndk/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -MMD -MP -MF /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/obj/local/arm64-v8a/objs-debug/SDL2/src/SDL_dataqueue.o.d -target aarch64-none-linux-android21 -fdata-sections -ffunction-sections -fstack-protector-strong -funwind-tables -no-canonical-prefixes  --sysroot /opt/android-ndk/toolchains/llvm/prebuilt/linux-x86_64/sysroot -g -Wno-invalid-command-line-argument -Wno-unused-command-line-argument  -D_FORTIFY_SOURCE=2 -fpic -O0 -UNDEBUG -fno-limit-debug-info  -I/home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni/SDL/include -I/opt/android-ndk/sources/android/cpufeatures -I/home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni/SDL   -DANDROID -DGL_GLEXT_PROTOTYPES -Wall -Wextra -Wdocumentation -Wdocumentation-unknown-command -Wmissing-prototypes -Wunreachable-code-break -Wunneeded-internal-declaration -Wmissing-variable-declarations -Wfloat-conversion -Wshorten-64-to-32 -Wunreachable-code-return -Wshift-sign-overflow -Wstrict-prototypes -Wkeyword-macro -Wno-unused-parameter -Wno-sign-compare -Wformat -Werror=format-security  -c  /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni/SDL/src/SDL_dataqueue.c -o /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/obj/local/arm64-v8a/objs-debug/SDL2/src/SDL_dataqueue.o


...


[arm64-v8a] Compile++      : harfbuzz <= hb-ft.cc
rm -f /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/obj/local/arm64-v8a/objs-debug/harfbuzz/src/hb-ft.o
/opt/android-ndk/toolchains/llvm/prebuilt/linux-x86_64/bin/clang++ -MMD -MP -MF /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/obj/local/arm64-v8a/objs-debug/harfbuzz/src/hb-ft.o.d -target aarch64-none-linux-android21 -fdata-sections -ffunction-sections -fstack-protector-strong -funwind-tables -no-canonical-prefixes  --sysroot /opt/android-ndk/toolchains/llvm/prebuilt/linux-x86_64/sysroot -g -Wno-invalid-command-line-argument -Wno-unused-command-line-argument  -D_FORTIFY_SOURCE=2 -fno-exceptions -fno-rtti -fpic -O0 -UNDEBUG -fno-limit-debug-info  -I/home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni/SDL2_ttf/external/harfbuzz/ -I/home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni/SDL2_ttf/external/harfbuzz/src/ -I/home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni/SDL2_ttf/external/harfbuzz/../freetype/include/ -I/home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni/SDL2_ttf/external/harfbuzz    -DANDROID -DHAVE_CONFIG_H -fPIC -Wformat -Werror=format-security   -c  /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni/SDL2_ttf/external/harfbuzz/src/hb-ft.cc -o /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/obj/local/arm64-v8a/objs-debug/harfbuzz/src/hb-ft.o
/home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni/SDL2_ttf/external/harfbuzz/src/hb-ft.cc:759:73: error: cast from 'void (*)(FT_Face)' (aka 'void (*)(FT_FaceRec_ *)') to 'FT_Generic_Finalizer' (aka 'void (*)(void *)') converts to incompatible function type [-Werror,-Wcast-function-type-strict]
  if (unlikely (!ft_face->generic.data || ft_face->generic.finalizer != (FT_Generic_Finalizer) hb_ft_face_finalize))
                                                                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni/SDL2_ttf/external/harfbuzz/src/hb.hh:263:25: note: expanded from macro 'unlikely'
#define unlikely(expr) (expr)
                        ^~~~
/home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni/SDL2_ttf/external/harfbuzz/src/hb-ft.cc:765:34: error: cast from 'void (*)(FT_Face)' (aka 'void (*)(FT_FaceRec_ *)') to 'FT_Generic_Finalizer' (aka 'void (*)(void *)') converts to incompatible function type [-Werror,-Wcast-function-type-strict]
    ft_face->generic.finalizer = (FT_Generic_Finalizer) hb_ft_face_finalize;
                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/jni/SDL2_ttf/external/harfbuzz/src/hb-ft.cc:1035:32: error: cast from 'void (*)(FT_Face)' (aka 'void (*)(FT_FaceRec_ *)') to 'FT_Generic_Finalizer' (aka 'void (*)(void *)') converts to incompatible function type [-Werror,-Wcast-function-type-strict]
  ft_face->generic.finalizer = (FT_Generic_Finalizer) _release_blob;
                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
3 errors generated.
make: *** [/opt/android-ndk/build/core/build-binary.mk:415: /home/xou/.local/share/python-for-android/build/bootstrap_builds/sdl2/obj/local/arm64-v8a/objs-debug/harfbuzz/src/hb-ft.o] Error 1


  STDERR:

```

--- a/pythonforandroid/recipes/sdl2_ttf/__init__.py
+++ b/pythonforandroid/recipes/sdl2_ttf/__init__.py
@@ -2,7 +2,7 @@ from pythonforandroid.recipe import BootstrapNDKRecipe
 
 
 class LibSDL2TTF(BootstrapNDKRecipe):
-    version = '2.20.2'
+    version = '2.22.0'
     url = 'https://github.com/libsdl-org/SDL_ttf/releases/download/release-{version}/SDL2_ttf-{version}.tar.gz'
     dir_name = 'SDL2_ttf'
 

